front/cypress/e2e/report_builder/idea_template.cy.ts

```diff
@@ -148,51 +153,20 @@ describe('Idea template', () => {
         cy.get('.e2e-text-box').eq(2).should('contain.text', text);
 
         // Save report
-        cy.intercept('PATCH', `/web_api/v1/reports/${reportId}`).as(
-          'saveReportLayout'
-        );
         cy.get('#e2e-content-builder-topbar-save > button').click({
           force: true,
         });
-        cy.wait('@saveReportLayout');
-
-        // Refresh page
-        cy.reload();
-
-        // Validate that the edited text is still there
-        cy.get('.e2e-text-box').eq(2).should('contain.text', text);
+        cy.wait('@saveReportLayout').then(() => {
+          // Refresh page
+          cy.reload();
 
-        // Remove report
-        cy.apiRemoveReportBuilder(reportId);
-      });
-    });
-
-    it.skip('autosaves report created from template', () => {
-      cy.apiCreateReportBuilder().then((report) => {
```

Comment:
These two "autosave" tests are incredibly flaky. I spent a lot of time trying to stabilize them with no luck :thinking: I also don't know how useful/vital they are? I've removed them for now since I don't feel they are super vital, but let me know if you feel differently :)